src/Backend.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CaT\Libs\TableProcessing;
+
+/**
+ * Backend interface for table processing.
+ * Defines options how to act with table elements
+ *
+ * @author Stefan Hecken 	<stefan.hecken@concepts-and-training.de>
+ */
+interface Backend
+{
+	/**
+	 * Delete the option in record
+	 */
+	public function delete(array $record);
```

Comment:
Finde das so in den Kommentaren blöd. Kann man nicht irgendwie beschreiben was da in dem array steckt?